.github/workflows/ci.yml

```diff
@@ -21,10 +21,14 @@ jobs:
       - run: poetry install
       - name: Run CLI tests
         run: poetry run make test
-      # Taken from: https://github.com/orgs/community/discussions/27149#discussioncomment-3254829
-      - name: Set path for candle and light
-        run: echo "C:\Program Files (x86)\WiX Toolset v3.14\bin" >> $GITHUB_PATH
-        shell: bash
+      - name: Set up .NET CLI environment
+        uses: actions/setup-dotnet@v4
+        with:
+          dotnet-version: "8.x"
+      - name: Install WiX Toolset
+        run: dotnet tool install --global wix --version 5.0.1
```

Comment:
We could set the version in an environment variable would that be enough?

```yaml
  env:
    wix_version: 5.0.1
- name: Install WiX Toolset
   run: dotnet tool install --global wix --version $wix_version
- name: Add WiX UI extension
   run: wix extension add --global WixToolset.UI.wixext/$wix_version
```

Reply:
It's not the duplication that worries me, but whether a future 5.1 Wix release breaks something. I'll give it a shot locally.

Reply:
Oops, yes, I started looking into the wrong thing. We could set the WiX version with a matrix stradegy. Something like:

```yaml
strategy:
      matrix:
        wix_version: [5.0.0, 5.0.1]

- name: Install WiX Toolset
   run: dotnet tool install --global wix --version ${{ matrix.wix_version }}
- name: Add WiX UI extension
   run: wix extension add --global WixToolset.UI.wixext/${{ matrix.wix_version }}
```

Reply:
Actually, I was thinking of something like the following:

```
dotnet tool install --global wix --version 5.*
wix extension add --global WixToolset.UI.wixext/5
```

I just realized though that there's an open issue for the latter, and it won't fetch the latest 5.x.y release of WixToolset.UI.wixext (see https://github.com/wixtoolset/issues/issues/8033)

Anyway, we can pin the extension version for now, but my suggestion is to always install the latest WiX version using `dotnet tool install`, if you agree.

Reply:
Right, now I got it. I hardcoded the version to avoid a situation where the windows installer accidentally gets built with a newer version of WiX Toolset and breaks in unexpected ways. E.g See the whole Scope debacle. 

We could drop the version numbers in CI to catch any regressions, but keep them in the build instructions to prevent accidentally upgrading WiX, fox example. I'm not particularly opinionated on this.

Reply:
And on that note, should I also remove the dotnet version? The setup-dotnet action picks whatever is the newest version installed in the runner if its not defined. I picked 8 since that's the version you'll get offered when downloading .NET SDK.

Reply:
> Or maybe even have two jobs in CI. One installs WiX without a version number to test for regressions and the other uses whichever version is used to build the current released installer and gets bumped when the installer is upgraded to a newer version of WiX

Excellent, that's the best of both worlds. Our CI is undergoing a major change, so no need to do this in this PR. I'll create a separate issue to track it.